test/integration/error-include-ignore-unsupported-files/expected-output.txt

```diff
@@ -1,2 +1 @@
-[snowpack] ignoring unsupported file "src/c.abcdefg"
```

Comment:
This was the only test change: now we don’t throw a warning; we just silently skip it.

Reason is that we were accidentally scanning a lot more text files than intended `.json`, `.txt`, but do we surface to the user that we are skipping those? Instead, this PR moves this message to a debug message visible with `--verbose`.